schemas/+optogenetics/OptogeneticSession.m

```diff
@@ -0,0 +1,18 @@
+%{
+# Information of a optogenetic session
+->acquisition.SessionTest
+---
+->reference.BrainArea
+hemisphere			               : enum('Left','Right','Bilateral')
+laser_nm			               : int                             # Laser color in nm
+laser_power                        : int                             # Laser power in (mW)
+frequency			               : float
+%}
```

Comment:
Do we need something about the waveform?

Reply:
It can be useful. I will ask opto people.
 I will try to gather up with all people performing Opto experiments and check if they have other ideas/suggestions for these fields.